jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -10,13 +10,81 @@
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import (PluginTemplateMixin, ViewerSelect, LayerSelect,
                                         PlotOptionsSyncState)
+from jdaviz.core.user_api import PluginUserApi
 from jdaviz.core.tools import ICON_DIR
 
 __all__ = ['PlotOptions']
 
 
 @tray_registry('g-plot-options', label="Plot Options")
 class PlotOptions(PluginTemplateMixin):
+    """
+    The Plot Options Plugin gives access to per-viewer and per-layer options and enables
+    setting across multiple viewers/layers simultaneously.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``multiselect``:
+      whether ``viewer`` and ``layer`` should both be in multiselect mode.
+    * ``viewer`` (:class:`~jdaviz.core.template_mixin.ViewerSelect`):
+    * ``layer`` (:class:`~jdaviz.core.template_mixin.LayerSelect`):
+    * :meth:`select_all`
+    * ``layer_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`)

```

Comment:
I agree this is confusing, but is inherited from glue (although if further naming tweaks could help, we can definitely do that).  Basically an image layer has the global layer visibility and then individual `image` and `contour` toggles.  We could hide the layer level toggle for image layers, I suppose, and require the user setting the two halves separately and listen to either being set to True to set the underlying glue state for the layer to True?  But we also need to think how this plays with non-image layers.

Reply:
Thinking on this more... `image_visible` probably goes better with `contour_visible` and `subset_visible`? So, we hide `layer_visible`? If not, we need to really explain what all these visibility toggles are for.

Reply:
I can look into adding extra logic to abstract it away, but I'm not sure it will be feasible, our hands are somewhat tied by the glue design here and have to make sure to expose enough flexibility that the user can't get stuck from blinking, etc.

Reply:
Abstracting probably is too complicated. Copious doc might be the easiest if you want to keep them all.

Reply:
I gave this quite a bit of thought and decided to take a stab at abstracting it away.  From the plot options API and UI, this removes `layer_visible` entirely and renames it to `line_visible` for the cases of a line layer.  Please try in both the UI and API and let me know if this feels more intuitive or is a step backwards.